control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1618,6 +1618,15 @@ func (r *HostedControlPlaneReconciler) reconcileKubeAPIServer(ctx context.Contex
 		return fmt.Errorf("failed to reconcile authentication token webhook config: %w", err)
 	}
 
+	pdb := manifests.KASPodDisruptionBudget(hcp.Namespace)
+	if result, err := controllerutil.CreateOrUpdate(ctx, r, pdb, func() error {
```

Comment:
shouldn't this be using our own `CreateOrUpdate`?

Reply:
Good catch, fixed